### PR TITLE
Pep8

### PR DESCRIPTION
**Pull request by WilliamZekaiWang**

### CODE REVIEW

**Comment by GopiGugan** [comment]
@WilliamZekaiWang to attempt to resolve merge conflicts. Once complete, we will do a dry run on BEVi

File: covizu/beadplot.py [hunk 1]
```diff
@@ -233,13 +253,13 @@ def parse_args():
     else:
         try:
             ctree = Phylo.read(args.tree, 'newick')
-        except:
-            print("Detected multiple trees in file, handling as bootstrap")
+        except ValueError as e:
```
Comment (WilliamZekaiWang) [inline covizu/beadplot.py:256]: Not sure if this is the right error catch

File: tests/test_gisaid_utils.py [hunk 2]
```diff
@@ -1,83 +1,143 @@
 import unittest
-from covizu.utils.gisaid_utils import *
+from covizu.utils.gisaid_utils import load_gisaid, batch_fasta, extract_features, sort_by_lineage
 
 
-def callback(s):
-    print(s)
+def callback(stuff):
```
Comment (WilliamZekaiWang) [inline tests/test_gisaid_utils.py:5]: Stuff is probably not the right variable name

Comment (ArtPoon) [reply tests/test_gisaid_utils.py:5]: For callback functions I usually use `msg`

File: mut_annotation_edit.py [hunk 3]
```diff
@@ -7,181 +10,218 @@
...
+            with open(qualified_txt_file, 'r', encoding='utf-8') as text_file:
+                lines = text_file.readlines()
+                generate_constellation_keys(
+                    lines,
+                    gene,
+                    qualified_txt_file,
+                    short_effect_desc,
+                    constellations_to_categories)
+                text_file.close()
+        except FileNotFoundError as err:
```
Comment (WilliamZekaiWang) [inline mut_annotation_edit.py:176]: I think this can also be a permission error?

File: covizu/clustering.py [hunk 4]
```diff
@@ -31,38 +32,39 @@ def recode_features(records, callback=None, limit=10000):
...
 
     # generate union of all features
     if callback:
-        callback("Reduced to {} variants; generating feature set union".format(len(fvecs)))
-    union = {}
-    labels = {}
-    indexed = []
+        callback(
+            f"Reduced to {len(fvecs)} variants; generating feature set union")
+    intermed_out = [{}, {}, []] # [{union}, {labels}, [idx]]
```
Comment (WilliamZekaiWang) [inline covizu/clustering.py:53]: I did this to lower the amount of local variables, but I feel like it hurts readability more than anything

Comment (WilliamZekaiWang) [inline covizu/clustering.py:53]: I did this to lower the amount of local variables, but I feel like it hurts readability more than anything

Comment (ArtPoon) [reply covizu/clustering.py:53]: I would rather keep the named local variables - can we increase the threshold instead?

**Comment by ArtPoon** [comment]
Do a dry run following PR to see if anything in `dev` has been broken

**Comment by WilliamZekaiWang** [comment]
Should be good to merge into dev for the dry run

**Pull request merged**